src/index.css

```diff
@@ -0,0 +1,5 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
```

Comment:
- [ ] Kindly consider following suggested font-family in the design guidelines.
the suggested fonts are `Lato` and `gill sans`. Lato is easily available on google fonts. and you can use similar looking font for gill sans if you dont find it easily.
![p](https://github.com/emhamza/Hotels-Archive/assets/96683843/bb590d9d-3f97-4892-b194-20610ba12254)